{"name": "test_sign_up_invalid_phone_num1", "status": "passed", "description": "the test is verify, that user can't sign up with invalid phone number", "start": 1680348730462, "stop": 1680348731165, "uuid": "79100b46-afbf-4958-bdbb-02965a30f771", "historyId": "ddb85d6b50ba6804f7489a08b6381fab", "testCaseId": "042bd5704a47a5e9a1eea8d4972e16aa", "fullName": "tests.chromeTests.testSignUpModal.TestSignUpModal#test_sign_up_invalid_phone_num1", "labels": [{"name": "parentSuite", "value": "tests.chromeTests"}, {"name": "suite", "value": "testSignUpModal"}, {"name": "subSuite", "value": "TestSignUpModal"}, {"name": "host", "value": "krot"}, {"name": "thread", "value": "12616-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.chromeTests.testSignUpModal"}]}